docs/stencil-docs/page-builder/product-widget-tutorial.md

```diff
@@ -1,50 +1,34 @@
-# Product Widget Tutorial
+# Create Widgets Powered by GraphQL
 
 <div class="otp" id="no-index">
 
 ### On this page
 - [Prerequisites](#prerequisites)
-- [Create a product widget](#create-a-product-widget)
-- [Widget preview](#widget-preview)
+- [Create the widget template](#create-the-widget-template)
+- [Place the widget using Page Builder](#place-the-widget-using-page-builder)
+- [Place the widget using the API](#place-the-widget-using-the-api)
 - [Resources](#resources)
 
 </div>
 
-Widgets are configurable and reusable components of content that merchants can display on their storefront. Widgets consist of a combination of HTML, JavaScript, CSS, and Handlebars, and are rendered as part of the storefront’s HTML. 
```

Comment:
I would leave this sentence in the tutorial. I think it gives developers a quick overview of BC's definition of a widget without overcrowding the document.

Reply:
If an "understanding of widgets" is a prerequisite for the tutorial, why do we need to define it in the tutorial?